db/re/item_db_usable.yml

```diff
@@ -38268,7 +38670,8 @@ Body:
       NoMail: true
       NoAuction: true
     Script: |
-      getitem 14591,30;
+      getitem 14591,500;
+      getitembound 23919,11,Bound_Account;
```

Comment:
The item give 30 siege map teleport scrolls according to kro description, please revert